spec/support/webmock.rb

```diff
@@ -2,18 +2,25 @@
 [:delete, :get, :post, :put, :patch].each do |method|
   self.class.send(:define_method, "a_#{method}") do |path, *args|
     opts = args.last.is_a?(Hash) ? args.pop : {}
+
     ver = args.first || opts[:api_version] || 1
     fmt = opts[:api_format] || :json
-    a_request(method, Gemfury.endpoint + path).with(
-      :headers => {
-        'Accept' => "application/vnd.fury.v#{ver}+#{fmt}",
-        'X-Gem-Version' => Gemfury::VERSION
-      }
-    )
+    endpoint = opts[:endpoint] || Gemfury.endpoint
```

Comment:
So a bit of a style nit pick - I usually try to keep the definition of a variable as close as possible to where I use it.  For example, here, I'd put the `endpoint` line right above the `a_request` call.  This helps readability for both `endpoint` definition/usage, and  definitions of `ver` and `fmt` move closer to where they're used for `Accept` header.